services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/SlotProcessor.java

```diff
@@ -101,7 +103,7 @@ public void setCurrentSlot(final UInt64 slot) {
 
   public void onTick(final UInt64 currentTime) {
     final UInt64 genesisTime = recentChainData.getGenesisTime();
-    if (currentTime.compareTo(genesisTime) < 0) {
+    if (currentTime.isLessThan(genesisTime)) {
```

Comment:
Nice. These methods are always so much easier to read than compareTo so good to switch over to them as we go.